charts/ai-software-templates/chatbot/0.1.0/templates/tests/test-connection-app.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "{{ .Release.Name }}-test"
+  annotations:
+    "helm.sh/hook": test
```

Comment:
I'm not familiar with the test helm hook lifecycle, do we need to wait for the service to become available, or will the test run after the service has become available?

Reply:
I think it runs for `helm install` only when the test flag is present (from their docs https://helm.sh/docs/topics/charts_hooks/#the-available-hooks). It runs for sure everytime we run `helm test` (and is the way that the [charts verifier](https://github.com/redhat-certification/chart-verifier) uses it)